lib/couchbase-orm/proxies/collection_proxy.rb

```diff
@@ -0,0 +1,53 @@
+require "couchbase"
+
+module CouchbaseOrm
+    class CollectionProxy
+
+        def get!(id, **options)
+            @proxyfied.get(id, Couchbase::Options::Get.new(**options))
```

Comment:
we should raise custom couchbase orm error here

Reply:
nice to have https://github.com/Mapotempo/couchbase-orm/issues/41